ast/parser.go

```diff
@@ -1956,12 +1957,25 @@ func (p *Parser) validateDefaultRuleValue(rule *Rule) bool {
 }
 
 type rawAnnotation struct {
-	Scope   string                `json:"scope"`
-	Schemas []rawSchemaAnnotation `json:"schemas"`
+	Scope            string                `yaml:"scope"`
+	Title            string                `yaml:"title"`
+	Description      string                `yaml:"description"`
+	Organizations    []string              `yaml:"organizations"`
+	RelatedResources []string              `yaml:"related_resources"`
+	Authors          []string              `yaml:"authors"`
```

Comment:
Optionally, we could make this a `[]interface{}`, to allow the user to do both
```
authors: 
 - John Doe <john@example.com>
```
and
```
authors:
 - name: John Doe
   email: john@example.com
```